runtime/reference/wasm.md

```diff
@@ -283,6 +283,19 @@ Rust crates useful. `web_sys` contains bindings to most of the Web APIs that are
 available in Deno, while `js_sys` provides bindings to JavaScript's standard,
 built-in objects.
 
+## Using wasmbuild for Rust WebAssembly in Deno
+
+[wasmbuild](https://github.com/denoland/wasmbuild) is an official Deno tool that
+simplifies working with Rust and WebAssembly in Deno projects. It automates the
+process of compiling Rust code to WebAssembly and generating TypeScript
+bindings, making it easy to call Rust functions from JavaScript.
+
+wasmbuild generates TypeScript definitions for your Rust functions, providing
+full type checking. The generated JavaScript can be used with bundlers like
+esbuild. Generated files can be committed directly to source control for easy
+deployment, and wasmbuild can watch your source files and rebuild when changes
+are detected.
```

Comment:
```suggestion
esbuild.
```

I'd remove these two parts because the generated files don't necessarily have to be committed to source control and we haven't implemented watch for Wasmbuild.